drivers/clock_control/clock_control_nrf.c

```diff
@@ -210,7 +210,8 @@ static void lfclk_start(void)
 
 static void lfclk_stop(void)
 {
-	if (IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION)) {
+	if (IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION)
+	    && !IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_FORCE_ALT)) {
```

Comment:
Everywhere this style is used:
```suggestion
	if (IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION) &&
	    !IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_FORCE_ALT)) {
```

Reply:
Fixed